Install/Install.cmake

```diff
@@ -63,6 +63,8 @@ install_targets(UrlLib)
 # ----------------
 # Core
 # ----------------
+install_include_for_targets(Foundation)
```

Comment:
Babylon/Api.h was added to a new CMake header-only interface project in the JsRuntimeHost repo. This the CMake install target change needed for installing Babylon/Api.h along with the other required headers.